frontend/src/features/projects/dispose/ProjectDisposeLayout.tsx

```diff
@@ -134,7 +134,6 @@ const ProjectDisposeLayout = ({ match, location }: { match: Match; location: Loc
           {getProjectRequest?.isFetching !== true ? (
             <Container fluid className="step-content">
               <Switch>
-                )}
```

Comment:
Is this a bug fix for a separate issue? This frontend file doesn't seem related to the PR work...